regulation-worker/internal/service/service.go

```diff
@@ -62,6 +62,9 @@ func (js *JobSvc) JobSvc(ctx context.Context) error {
 	deletionStart := time.Now()
 
 	status = js.Deleter.Delete(ctx, job, destDetail)
+	if status == model.JobStatusFailed && job.FailedAttempts >= 4 {
```

Comment:
The maximum number of failed attempts could be configurable with default value 5 (4+1)

Reply:
done